packages/kit-bg/src/vaults/impls/aptos/Vault.ts

```
@@ -844,12 +844,17 @@ export default class VaultAptos extends VaultBase {
-    return {
+    const params = {
       encodedTx: {
-        ...(encodedTx as object),
+        ...encodedTx,
+        ...encodedTx.payload,
         rawSignTx,
-      } as unknown as IEncodedTx,
+      },
     };
+
+    delete params.encodedTx.payload;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)